plugin/init.lua

```diff
@@ -178,7 +178,7 @@ function M.apply_to_config(config, _)
   end
 
   local keys = {
-    { key = config.leader.key, mods = "LEADER|" .. config.leader.mods, action = act.SendKey(config.leader) },
+    { key = config.leader.key, mods = "LEADER|" .. config.leader.mods, action = act.SendKey({ key = config.leader, mods = config.leader.mods }) },
```

Comment:
```suggestion
    { key = config.leader.key, mods = "LEADER|" .. config.leader.mods, action = act.SendKey({ key = config.leader.key, mods = config.leader.mods }) },
```

Throws error otherwise because config.leader is not a string.